cairo/src/Slay/Cairo.hs

```diff
@@ -122,5 +122,5 @@ instance RenderElement g (PrimCurve g) where
 instance RenderElement g (PrimCircle g)   where
   renderElement (PrimCircle cc c) getG (Offset x y) = do
       setSourceColor $ getG cc
-      Cairo.arc (toSigned c + x) (toSigned c + y) (toSigned c) 0 180
+      Cairo.arc (toSigned c + x) (toSigned c + y) (toSigned c) 0 (2 * pi)
```

Comment:
180 degrees is π/2, not 2π

Reply:
A circle is 2pi

Reply:
Okay, I thought you needed a half-circle judging by the original code.